plugins/chat/config/locales/client.en.yml

```diff
@@ -323,8 +323,8 @@ en:
         channel_wide_mentions_label: "Allow @all and @here mentions"
         channel_wide_mentions_description: "Allow users to notify all members of #%{channel} with @all or only those who are active in the moment with @here"
         auto_join_users_label: "Automatically add users"
-        auto_join_users_info: "Check hourly which users have been active in the last 3 months and, if they have access to the %{category} category, add them to this channel."
-        enable_auto_join_users: "Automatically add all recently active users"
+        auto_join_users_info: "Check hourly which users have been active in the last 3 months. Add them to this channel if they have access to the %{category} category."
+        auto_join_users_info_no_category: "Check hourly which users have been active in the last 3 months. Add them to this channel if they have access to the selected category."
```

Comment:
Im mixed on this, but this might be hard to translate otherwise.

Reply:
Yes, it's a very good decision to split this into two strings! :+1: